File: cpp/8kyu/bonus_time.cpp
#include <string>
using namespace std;

string bonus_time(int salary, bool bonus)
{
    string res{"$"};
    if (bonus)
    {
        res += to_string(salary * 10);
    }
    else
    {
        res += to_string(salary);
    }
    return res;
}

File: go/8kyu/bonus_time.go
package kata

import (
	"fmt"
	"strings"
)

func BonusTime(salary int, bonus bool) string {
	var sb strings.Builder
	sb.WriteString("£")
	if bonus {
		fmt.Fprintf(&sb, "%v", salary*10)
	} else {
		fmt.Fprintf(&sb, "%v", salary)
	}
	return sb.String()
}
